From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 3 Jan 2019 19:37:07 -0500
Subject: universal-access: Remove redundant label-for relations

The appropriate relation is already set with the mnemonic_widget property.

In particular, this also fixes an issue where a screenreader would
emit "Crosshairs" every time a new tab was entered in the Zoom dialog
because of some incorrect labelled-by relations.

Applied-upstream: 3.31.90, commit:0f92ffb01afa3fb9337c9fa98f359bef499c4c9a
---
 panels/universal-access/zoom-options.ui | 24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/panels/universal-access/zoom-options.ui b/panels/universal-access/zoom-options.ui
index 85723e0..baeec2f 100644
--- a/panels/universal-access/zoom-options.ui
+++ b/panels/universal-access/zoom-options.ui
@@ -131,9 +131,6 @@
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="use_action_appearance">False</property>
-                <accessibility>
-                  <relation type="labelled-by" target="xhairs-section-heading"/>
-                </accessibility>
               </object>
               <packing>
                 <property name="expand">False</property>
@@ -582,9 +579,6 @@
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">crosshair_length_slider</property>
                                     <property name="width_chars">12</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="crosshair_length_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -722,9 +716,6 @@
                                 <property name="visible">True</property>
                                 <property name="can_focus">True</property>
                                 <property name="use_action_appearance">False</property>
-                                <accessibility>
-                                  <relation type="labelled-by" target="xhairs-section-heading"/>
-                                </accessibility>
                               </object>
                               <packing>
                                 <property name="expand">False</property>
@@ -835,9 +826,6 @@
                                     <property name="justify">right</property>
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">inverse_enabled_switch</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="inverse_enabled_switch"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -859,9 +847,6 @@
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">brightness_slider</property>
                                     <property name="width_chars">12</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="brightness_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -882,9 +867,6 @@
                                     <property name="justify">right</property>
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">contrast_slider</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="contrast_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -905,9 +887,6 @@
                                     <property name="justify">right</property>
                                     <property name="use_underline">True</property>
                                     <property name="mnemonic_widget">grayscale_slider</property>
-                                    <accessibility>
-                                      <relation type="label-for" target="grayscale_slider"/>
-                                    </accessibility>
                                     <attributes>
                                       <attribute name="scale" value="1.25"/>
                                     </attributes>
@@ -996,9 +975,6 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="use_action_appearance">False</property>
-                                        <accessibility>
-                                          <relation type="labelled-by" target="xhairs-section-heading"/>
-                                        </accessibility>
                                       </object>
                                       <packing>
                                         <property name="expand">False</property>
